src/util/PortfolioUploadMediaService.js

```diff
@@ -0,0 +1,159 @@
+import {
+  deletePortfolioMedia,
+  deleteWelcomeVideo,
+  getUserData,
+  uploadPortfolioMedia,
+  uploadWelcomeVideo
+} from './DataBaseRequests';
+
+import { useAuth } from '../AuthProvider';
+import { useState } from 'react';
+
+const useMediaUploader = (mediaType) => {
+  const [showFileInput, setShowFileInput] = useState(false);
+  const [uploadedFiles, setUploadedFiles] = useState([]);
```

Comment:
Should we reset the state after successful upload or are we sure that there won't be any residual files that will be uploaded again?

Reply:
Ok. I'll check this state.

Reply:
I checked, and indeed the file data remains in the state. But due to the fact that the form is updated and the upload button is blocked until a new file is selected, the old file is not sent. I added resetting this state anyway. Thank you for your comment.

Reply:
Good catch! Better to be safe then sorry.